src/scss/config/_layout.scss

```diff
@@ -28,3 +29,14 @@ $z-index: ('bump', 'dropdown', 'skip-link');
     transform: translateY(calc(var(--offset) * -1));
   }
 }
+
+// Breakpoints
+@each $name, $size in meta.module-variables('page-sizes') {
```

Comment:
Looking at the whole list of page-sizes below, I'm not sure it makes sense to turn them all into breakpoints? Some of them are things like logo dimensions.

Reply:
@mirisuzanne I moved `logo-item` and `grid-item` to theme-sizes in https://github.com/oddbird/oddleventy/pull/306/commits/6bf56ceeaf8da19b0c67605c9341586443c3d6ae.

And there was a card for relocating the `extra` and `main-min` variables that I figured would be easier knock out here too. I don't think we are using those sizes as breakpoints either. Do you think they make sense in `theme-sizes` as well? see https://github.com/oddbird/oddleventy/pull/306/commits/2410ea816c4602d741265868ccf969fb96a9c964